Litton2008 Pull Request

laylakalandjian opened the pull request:
Added Litton2008 rescue folder

laylakalandjian committed:
Townsend1995

new data rescue folder for Townsend1995 with completed lvl 0 data read. Includes Townsend1995.bib, Townsend1995_Methods.bib, README_Townsend1995.qmd, Townsend1995_Methods.md, Townsend1995_Table1.csv, Townsend1995_Table2.csv for a total of 6 files.

https://github.com/ktoddbrown/SoilDRaH/issues/74

laylakalandjian committed:
Added all files to folder

Townsend1995.bib, Townsend1995_Methods.bib, Townsend1995_Methods.md, Townsend1995_Table1.csv, and Townsend1995_Table2.csv

laylakalandjian committed:
Merge pull request #1 from laylakalandjian/laylakalandjian-patch-1

Townsend1995

laylakalandjian committed:
Update README_Townsend1995.qmd

Fixed file names in r chunks to match file names at beginning of README, fixed lvl 0 data read formatting

laylakalandjian committed:
Merge branch 'ktoddbrown:main' into main

laylakalandjian committed:
Create temp

laylakalandjian committed:
Added Litton2008 Files

Litton2008.bib, methods.bib, README, Table 2 csv, and Figure 6 csv.

laylakalandjian committed:
Delete data/rescue/Litton2008/temp

ktoddbrown commented on data/rescue/Litton2008/Litton2008.bib:
```diff
@@ -0,0 +1,15 @@
+@article{https://doi.org/10.1111/j.1365-2486.2008.01546.x,
```
use AUTHORYEAR for the bib key here instead of the doi

ktoddbrown commented on data/rescue/Litton2008/README_Litton2008.qmd:
```diff
@@ -0,0 +1,179 @@
+Below are the citation notes from the methods section.
+Some citations are missing, those are noted here and how best guess were generated.
+
+- [ ] short citation
```
You can remove this section of there are no citation notes.

ktoddbrown reviewed (changes requested):
You are missing the methods.md file. Some changes here but I don't think it will take you long.

laylakalandjian committed:
Added Litton2008 Methods Markdown File

was missing from original pull request

laylakalandjian committed:
Updated bib key to AUTHORYEAR from doi

laylakalandjian committed:
Re-uploaded Litton2008 README and Figure 6

Figure6 csv -- added additional column for letters indicating statistical significance, changed SD in column headers to SE (transcription error)
README -- removed blank citation notes section

laylakalandjian committed:
Litton2008 README Removed "Citation Notes" section + updated figure 6 description

laylakalandjian committed:
Updated transcription errors in Litton2008 Figure 6

corrected SD to SE -- transcription error
added column for letters denoting statistical significance

laylakalandjian committed:
Deleted extra temp file in Townsend1995

ktoddbrown reviewed (changes requested):
Should be pretty quick!

laylakalandjian committed:
Update README_Litton2008.qmd

Added level 0 data read and coded in figure 6

laylakalandjian committed:
Created temp folder

laylakalandjian committed:
Added temp files to folder

laylakalandjian committed:
Deleted placeholder file in temp folder

ktoddbrown reviewed (approved):
Looks good enough!
